### DISCUSSION TITLE
changing root/base directories with strider plugin

### DISCUSSION THREAD

**Discussion by johnnason** [zellij-org/zellij#401] [Q&A] [answered]
I might be being daft but I'm having trouble figuring out how to use strider effectively.
If I open zellij with the strider plugin it uses my current working directory. If I open a new zellij tab, I'm still bound to that same root directory ("left" nav does nothing).
I can navigate deeper into the hierarchy and back up to the "root" (where I launched from) but I can not change the root directory.

For instance, I might be in zellij working in a repo. I open a new tab and want to display the directory hierarchy of a different repo. Once strider is open I can not figure out how to get beyond its original root context .

 I'd like to start using this as my daily driver and with this plus detach support I'll have the core feature I need.

**Comment by a-kenji** [comment, marked as answer]
Hello! You are far from being daft! That is currently sadly not possible.
That is a feature/limitation of the current architecture and is indirectly already in the queue for fixing, if not necessarily very soon.
